README.md

```diff
@@ -1,138 +1,23 @@
-
-    ```bash
-    docker build -t messenger .
-    ```
-
-2. From the `messenger` repository, start the PostgreSQL database:
-
-    ```bash
-    docker-compose up -d
-    ```
-
-3. Start the `messenger` service in a container:
-
-    ```bash
-    docker run -d -p 8083:8083 --name messenger -e PGPASSWORD=postgres -e CREATE_DB_NAME=messenger -e PGHOST=messenger-db-1 -e AMQPHOST=rabbitmq -e AMQPPORT=5672 -e PORT=8083 --network mm_2023 messenger
-    ```
-
-4. SSH into the container to set up the PostgreSQL DB:
-
-    ```bash
-    docker exec -it messenger /bin/bash
-    ```
-
-5. Create the PostgreSQL DB:
-
-    ```bash
-    PGDATABASE=postgres node bin/create-db.mjs
-    ```
-
-6. Create the PostgreSQL DB tables:
-
-    ```bash
-    node bin/create-schema.mjs
-    ```
-
-7. Create some PostgreSQL DB seed data:
-
-    ```bash
-    node bin/create-seed-data.mjs
-    ```
+## Running just shared infrastructure
+If you are running some subset of the services manually, just run `docker-compose up` from this repository to start up the message queue.  This does not set up the NGINX load balancer.
 
-### Start the `notifier` Service
+## Setting up the whole demo architecture.
+Starting up the entire system including the applications is not part of the role of this repository. However, instructions are included here for simplicity.
 
-1. From the `notifier` repository, build the Docker image:
+We provide two methods to set things up:
+1. [Quick Setup](docs/quick-setup.md): Based on `docker-compose`. Use this one if you just want to get started quickly.
+1. [Manual Setup](docs/manual-setup.md): Uses some `docker-compose` but mostly raw docker commands. Use this if you want go through what the quick start does for you automatically for learning.
 
-    ```bash
-    docker build -t notifier .
-    ```
-
-2. From the `notifier` repository, start the PostgreSQL database:
-
-    ```bash
-    docker-compose up -d
-    ```
-
-3. Start the `notifier` service in a container:
-
-    ```bash
-    docker run -d -p 8084:8084 --name notifier -e PGPASSWORD=postgres -e CREATE_DB_NAME=notifier -e PGHOST=notifier-db-1 -e AMQPHOST=rabbitmq -e AMQPPORT=5672 -e PORT=8084 -e PGPORT=5433 --network mm_2023 notifier
-    ```
-
-4. SSH into the container to set up the PostgreSQL DB:
-
-    ```bash
-    docker exec -it notifier /bin/bash
-    ```
-
-5. Create the PostgreSQL DB:
-
-    ```bash
-    PGDATABASE=postgres node bin/create-db.mjs
-    ```
-
-6. Create the PostgreSQL DB tables:
-
-    ```bash
-    node bin/create-schema.mjs
-    ```
-
-7. Create some PostgreSQL DB seed data:
-
-    ```bash
-    node bin/create-seed-data.mjs
-    ```
-
-### Use the Service
-
-Follow the instructions [here](https://github.com/microservices-march/messenger#using-the-service) to test out the service. You should see notification logs coming from the `notifier:1` container.  You can see them easily by running `docker logs -f notifier`
+## NGINX (Load Balancer)
```

Comment:
```suggestion
## Components
### NGINX (Load Balancer)

```
It feels a bit weird to simply add NGINX and RabbitMQ at the end of the doc. If we group both under a Components header (or something similar) it will help make the structure of the doc clearer/more logical.